packages/flutter/lib/src/widgets/sliver.dart

```diff
@@ -250,6 +261,21 @@ class SliverChildBuilderDelegate extends SliverChildDelegate {
   /// Defaults to true.
   final bool addRepaintBoundaries;
 
+  /// Whether to wrap each child in an [IndexedChildSemantics].
+  ///
+  /// Defaults to true.
+  final bool addSemanticIndexes;
```

Comment:
This property reads like a method name. `isAddSemanticIndexes`? Also, do we prefer "indexes" or "indices"? We should probably be consistent with the rest of the API.

Reply:
`ag -Q indices | wc -l` -> 36
`ag -Q indexes | wc -l` -> 47

Guess we are agnostic

Reply:
I think this matches the naming of the other config